test/sequential/test-repl-persistent-history.js

```diff
@@ -74,35 +74,41 @@ const oldHistoryPath = path.join(fixtures, 'old-repl-history-file.json');
 const tests = [{
   env: { NODE_REPL_HISTORY: '' },
   test: [UP],
-  expected: [prompt, replDisabled, prompt]
+  expected: [prompt, replDisabled, prompt],
+  event: 'end'
```

Comment:
> It makes me wonder if we should defer emitting close until after flushHistory though?

That's what I was thinking.

I can't find where the 'end' event is emitted on the repl/readline though..